jib-maven-plugin/scripts/prepare_release.sh

```diff
@@ -58,3 +58,4 @@ EchoGreen 'File a PR for the new release branch:'
 echo https://github.com/GoogleContainerTools/jib/pull/new/${BRANCH}
 
 EchoGreen "Merge the PR after the plugin is released."
+EchoGreen "Run './scripts/update_gcs_latest ${VERSION}' when the release is complete to update the latest version string on GCS."
```

Comment:
Q: I put the GCS update stuff in a separate script since the maven release process takes some time. Should we just combine the scripts anyway and not worry about GCS reporting an unrelease version for a few minutes/hours?

Reply:
I think we should update the version only after we verify a new release is available for consumption for both Gradle and Maven.

Add `.sh`.